temba/utils/analytics/analytics.py

```diff
@@ -60,47 +65,20 @@ def get_intercom_user(email):
 
 def gauge(event, value=None):  # pragma: no cover
     """
-    Triggers a gauge event in Librato
+    Triggers a gauge event in all metric backends
     """
-    if value is None:
-        value = 1
-
-    # settings.HOSTNAME is actually service name (like textit.in), and settings.MACHINE_NAME is the name of the machine
-    # (virtual/physical) that is part of the service
-    reporting_hostname = "%s.%s" % (settings.MACHINE_HOSTNAME, settings.HOSTNAME)
+    global _metric_backends
+    for backend in _metric_backends:
+        backend.gauge(event, value)
 
-    if _librato:
-        _librato.gauge(event, value, reporting_hostname)
 
-
-def identify_org(org, attributes=None):
```

Comment:
Is this being removed on purpose? We use intercom for support so we don't want that ripped out.

Reply:
I didn't see anywhere in the code that was calling this, so I removed it. Happy to put it back if it is actually needed.

Reply:
Huh, you may be right actually, this may all be happening in JS. @ericnewcomer can you confirm?

Reply:
If memory serves, this was used as part of a migration and is no longer needed.